src/archiefvernietigingscomponent/destruction/models.py

```diff
@@ -337,6 +337,7 @@ class DestructionListAssignee(models.Model):
         "accounts.User", on_delete=models.PROTECT, verbose_name=_("assignee"),
     )
     order = models.PositiveSmallIntegerField(_("order"))
+    assigned_on = models.DateTimeField(_("assigned on"), default=None, null=True,)
```

Comment:
```suggestion
    assigned_on = models.DateTimeField(_("assigned on"), blank=True, null=True,)
```

Some helpful explanation about blank/null: https://docs.djangoproject.com/en/3.2/ref/models/fields/#null

Please remove migrations 18-23 and regenerate one after you updated the model.